includes/REST_API/Products_Controller.php

```diff
@@ -144,15 +144,16 @@ public function get_items( $request ) {
 		$results = [];
 
 		// TODO(#11154): Refactor to extract product query logic out of this controller.
-		$shopify_host         = $this->settings->get_setting( Settings::SETTING_NAME_SHOPIFY_HOST );
-		$shopify_access_token = $this->settings->get_setting( Settings::SETTING_NAME_SHOPIFY_ACCESS_TOKEN );
+		$shopping_provider = $this->settings->get_setting( Settings::SETTING_NAME_SHOPPING_PROVIDER);
 
-		if ( ! empty( $shopify_host ) && ! empty( $shopify_access_token ) ) {
-			return $this->get_items_shopify( $request );
+		if( $shopping_provider === 'woocommerce' ){
```

Comment:
Instead of if statements, a switch statement might be cleaner here.

Reply:
Example 

```
switch ( $shopping_provider ){
    case 'woocommerce':
        $response = $this->get_items_woocommerce( $request );
        break;
     case 'shopify':
       $response = $this->get_items_shopify( $request );
       break;
   default:
     $response = rest_ensure_response( [] );
}

return $response;

Reply:
Sure sound good - updated